plugin/Dockerfile

```diff
@@ -1,16 +1,38 @@
-FROM node:16 AS build
+FROM  registry.access.redhat.com/ubi8/nodejs-16 AS build
+
+USER root
+RUN command -v yarn || npm i -g yarn
 
 ADD . /usr/src/app
 WORKDIR /usr/src/app
 RUN yarn install && yarn build
 
-FROM nginx:stable
+FROM registry.access.redhat.com/ubi8/nginx-120
+
+# Add application sources to a directory that the assemble script expects them
+# and set permissions so that the container runs without root access
+USER 0
+
+# Install upgrades security
+#   xz-libs RHSA-2022:4991
+#   urllib3 CVE-2020-26137 CVE-2021-33503  Not available in the ubi8
+#
+RUN yum -y upgrade xz-libs
 
-RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
-COPY --from=build /usr/src/app/dist /usr/share/nginx/html
+COPY --from=build /usr/src/app/dist /usr/share/nginx/html/
+RUN chown -R 1001:0 /usr/share/nginx/html/
 
-# Have the plugin log its version immediately upon startup
 ARG VERSION
 ARG COMMIT_HASH
-RUN echo "echo 'OpenShift Service Mesh Plugin: Version=[${VERSION}], Commit=[${COMMIT_HASH}]'" > /docker-entrypoint.d/01-log-version.sh
-RUN chmod ug+x /docker-entrypoint.d/01-log-version.sh
\ No newline at end of file
+RUN printf "\necho 'OpenShift Service Mesh Plugin: Version=[${VERSION}], Commit=[${COMMIT_HASH}]' >> /proc/1/fd/1" >> ${NGINX_CONTAINER_SCRIPTS_PATH}/common.sh
+
+USER 1001
+LABEL com.redhat.component="openshift-service-mesh-plugin" \
+      name="openshift-service-mesh-plugin" \
+      summary="OpenShift Service Mesh Kiali Plugin" \
+      description="Kiali plugin for Openshift Service Mesh" \
```

Comment:
"Kiali plugin for Openshift Service Mesh" -> "The OpenShift Service Mesh Plugin adds Kiali functionality to the OpenShift Console."

Wording for this effort should be synced.

The operator would have the "master" info about it.